Ajout de contenu pour les formations AGIR Transport

nlehuby opened the pull request:
Voici le contenu présenté lors de la session du 20 novembre 2025

TuThoThai committed:
Formation AGIR - Partie 1

TuThoThai committed:
correction lien

TuThoThai committed:
agenda

nlehuby committed:
ajout divers sur l'intro et les concepts de base

Henault committed:
Mise à jour du texte

Henault committed:
MaJ Texte

Remplacement de "ce matin" par "lors de la session précédente"

nlehuby committed:
ajouts sur l'accessibilité pour la formation AGIR

thbar commented:
Merci @nlehuby !

Je vois deux besoins :
1. À la fin de la formation AGIR Transport, générer des supports PDF spécifiques et leur envoyer
2. Conserver "ce qui est non spécifique AGIR Transport et peut être mergé dans main", le merger dans `main`, et faire une release de la formation principale

Est-ce que vous avez aussi ça en tête ? (@nlehuby @TuThoThai @Henault @adurand-cs) ?

Si oui du coup je vous propose:
1. On garde la branche `AGIR` pour le moment, on la met à jour, puis on envoie les PDF associés
2. Une fois que c'est fait, on fait (via une sous-branche de `AGIR`) des suppressions/rollback de tout ce qui n'intéresse pas `main` ; ça fait une 2ème PR "fille" (plus facile à relire au propre, en conservant les auteurs d'origine), qu'on mergera dans AGIR (la présente PR) une fois qu'on sera au propre, puis on mergera le tout dans `main`, et quand on fera les release notes de la nouvelle version, on créditera AGIR pour ces améliorations au passage.

Qu'en pensez-vous ?

thbar commented:
(ça ou un autre schéma équivalent je précise !)

nlehuby commented:
J'ai ajouté le contenu de la dernière session de formation ce jour, c'est ok pour moi pour transmettre les supports de formation aux participants.

Sur la partie accessibilité, il n'y a que des ajouts, qui restent pertinents et peuvent être mergés dans main. (À mon avis, il n'y a que le livebook d'intro qui nécessite des modifications pour main).

TuThoThai committed:
MàJ Readme

MàJ pour la formation AGIR

TuThoThai committed:
MàJ intro

Rajout logo AGIR, mise à jour des intervenants

TuThoThai committed:
Logo agir

TuThoThai committed:
Update schémas Transmodel-NeTEx-GTFS

Utilisation de schémas plus récents pour la visualisation Transmodel, NeTEx et GTFS

TuThoThai committed:
Ajout liens vers fichiers

Ajout des liens vers les fichiers XML figés pour les exercices

TuThoThai committed:
correction typo readme
